scripts/build.mjs

```diff
@@ -0,0 +1,50 @@
+import chalk from 'chalk'
+import { build } from 'esbuild'
+import { existsSync } from 'fs'
+import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill'
+import walk from 'walkdir'
+
+const ignorePackages = ['teleport-repl-component', 'teleport-test']
+const emitter = walk('packages', {
+  max_depth: 1,
+})
+
+emitter.on('directory', async (path) => {
+  const split = path.split('/')
+  const packageName = split[split.length - 1]
+  if (ignorePackages.includes(packageName)) {
+    return
+  }
+
+  await buildPackage(path, packageName)
+})
+emitter.on('error', (err) => console.log(err))
+
+export const buildPackage = async (path, packageName) => {
+  const entry = `${path}/src/index.ts`
+  const isEntryExists = existsSync(entry)
+
+  if (!isEntryExists) {
+    throw new Error(`Entry file missing from ${packageName}`)
+  }
+
+  await build({
+    entryPoints: [`${path}/src/index.ts`],
+    outdir: `${path}/dist/cjs`,
+    format: 'cjs',
+    plugins: [new NodeModulesPolyfillPlugin()],
```

Comment:
- [ ] Don't add polyfills in these builds, the end users djoukd be able to sdd them depending on the tsrger platform. Loke repl, where we polyfill path and fs explicitly.
- [ ] add build target as es6, anyways es5 is not supported by esbuild. But this helps in transpiling any higher js syntaxes to es6.
- [ ] Add export map in all packages